keyboards/idobao/id67/keymaps/vinorodrigues/keymap.c

```diff
@@ -121,15 +121,15 @@ void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
 
     if (host_keyboard_led_state().caps_lock) {
         if (isRGBOff) {
-            rgb_matrix_set_color(ID67_CAPS_LOCK_KEY_INDEX, v, v, v);  // white
```

Comment:
line 85 of this file should contain:
```
#define CAPS_LOCK_KEY_INDEX 36  // position of Caps Lock key
```

this value is specific to this keymap ... not sure how it ended up in root file `id67.c`

Reply:
Ah, sorry. I suppose I was thrown off by this comment 
[https://github.com/qmk/qmk_firmware/blob/master/keyboards/idobao/id67/id67.c#L22](https://github.com/qmk/qmk_firmware/blob/master/keyboards/idobao/id67/id67.c#L22)

Reply:
np.  it's confusing as heck.  i had to retest my assumptions twice to confirm what i'd written a few months back.

and hey!  thanks for trying to fix it - was was slow to see that it was posted and didn't check anyway.  by the time i realised you'd already pr'ed it.   no foul and the community thanks you for your contributions!